Formula/c/cmctl.rb

```diff
@@ -19,10 +19,11 @@ class Cmctl < Formula
   depends_on "go" => :build
 
   def install
+    project = "github.com/cert-manager/cmctl"
```

Comment:
```suggestion
    project = "github.com/cert-manager/cmctl/v2"
```

https://github.com/cert-manager/cmctl/blob/f5f4439e47f3092d4b6e05dd39ca4a115f8eef02/make/00_mod.mk#L15